src/core/issues-notes.js

```diff
@@ -203,6 +196,26 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (!issueSummaryElement) return;
+  const heading = issueSummaryElement.querySelector("h2, h3, h4, h5, h6");
+
+  issueList.hasChildNodes()
+    ? issueSummaryElement.append(issueList)
+    : issueSummaryElement.append(hyperHTML`<p>${l10n.no_issues_in_spec}</p>`);
+  heading ||
+    issueSummaryElement.insertBefore(
```

Comment:
Just using `if (!heading) {` would be cleaner. 

I still think there is a bug tho... you could have:

```HTML
<section id="issue-summary">
  <p>Issues are amazing</p>
  <section>
     <h3>Amazing</h3>
  </section>
</section>
```

(Make sure you add the above to a test)

So there is check needed: that the `heading !== issueSummaryElement.firstElementChild`.

Reply:
I wonder if `insertAdjacentHTML("afterbegin”, ...)` couldn’t be used here instead 🧐 if yes, then it’s good as it avoids having to use hyperHTML).